src/main/kotlin/frc/robot/subsystems/hood/HoodIO.kt

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.hood
+
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Voltage
+import org.team9432.annotation.Logged
+
+interface HoodIO {
+    val inputs: LoggedHoodInputs
+
+    fun updateInternalEncoder() {}
+
+    fun setAngle(angle: MutableMeasure<Angle>) {}
+
+    fun setAngle(angle: MutableMeasure<Angle>, torqueChassisCompensation: Double) {}
```

Comment:
I like the compensation!
1. The name should be a bit more general -- the hood doesn't care about the chassis, the intention is just to compensate for external forces on the hood (if I understand correctly). So the name should be more precise.
2. You need to use a supplier here, the chassis acceleration changes constantly. Also, this would enable you to remove the first method and use just an empty supplier if compensation is not desired.

Reply:
d7cad7e123bb857528329069e88cfa8f0a97545e

Reply:
We will remove the first method once we implement the compensation.